src/bin/prebuilt.ts

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
It's a ts file. Should use `ts-node`

Reply:
![image](https://user-images.githubusercontent.com/3842474/87855943-38744f00-c94e-11ea-97a5-c8d91942aed4.png)

Reply:
No need to compile it into JS, add complexity to the repo

Reply:
???????

Reply:
I think it's okay to require users (the only user is Maskbook) to install the ts-node to run it.

Reply:
I think that this was unreasonable.

Reply:
ok, so remove the esm part, only leave the cjs part. Or bundle these two commands as a single file by rollup

Reply:
You come to implements it

Reply:
Remove the esm part will be ok